src/components/panel/panel.js

```diff
@@ -254,34 +253,22 @@ angular
 
 /**
  * @ngdoc method
- * @name MdPanelRef#addClass
```

Comment:
Why not leave `addClass` et al here, since they're separately useful from show/hide?